### DISCUSSION TITLE
No module named 'packaging'

### DISCUSSION THREAD

**Discussion by alirezatondro** [idaholab/moose#25748] [Q&A Getting Started]
Hello,

I've been trying to install get moose to work on my M1 Macbook Air, but during the build of moose, I get this error:

```
Traceback (most recent call last):
  File "/Users/Alireza/projects/moose/framework/scripts/get_repo_revision.py", line 19, in <module>
    from packaging import version
**ModuleNotFoundError: No module named 'packaging'**
Checking if header needs updating: /Users/Alireza/projects/moose/framework/include/base/MooseRevision.h...

Failed to generate MooseRevision.h

make: *** [/Users/Alireza/projects/moose/framework/moose.mk:351: /Users/Alireza/projects/moose/framework/include/base/MooseRevision.h] Error 1
(moose) Alirezas-MacBook-Air:test Alireza$ 
```
Reinstalling packaging also did not seem to be helpful on this matter, any suggestions from you guys is appreciated. Thanks!

**Comment by GiudGiud** [comment]
hello

The problem is likely that the python found during the make process is not the one you installed packaging for.

You can either:
- install packaging on the system python
OR
- check the PATH environment variable to see why the wrong python is being found before the mamba python

Guillaume
